IndyAV/command/indyav_launch/launch/joydrive.launch

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<launch>
+
+ <!-- Joystick Driver -->
+  <node pkg="joy" type="joy_node" name="joy" output="screen">
+   <param name="dev" value="/dev/input/js0"/>
```

Comment:
could please add the `dev` param as a launch argument. You can see https://github.com/uf-mil/mil/blob/master/IndyAV/command/indyav_launch/launch/master.launch with argument `environment` for an example

Reply:
Added launch argument  `js` that defaults to "js0". The value of this argument is appended to "/dev/input/" and then used as the value for param `dev`.
For example, `roslaunch indyav_launch joydrive.launch js:=js1` will set param `dev` to "/dev/input/js1".